### ISSUE TITLE
Update to latest rector

### ISSUE THREAD

**Issue by goba** [drupal8-rector/drupal8-rector#35]
There is an explicit requirement for rector/rector 0.4.12, the current rector version is 0.6.2

**Comment by goba** [comment]
See https://github.com/drupal8-rector/drupal8-rector/pull/36

**Comment by mxr576** [comment]
Yes, the currently supported version is quite old, but it was locked to a specific version for a reason. Rector is dev only (0.x.x) currently, every new releases of it is expected to contain a breaking change for this library. Based on my previous experiences this was a good decision because updating to the next version of Rector was always challenging. Basically every new rector added to a new release had to be validated whether it is compatible with Drupal 8 or not, or it had to be configured for Drupal 8.
It seems the last time when I tried to update Rector was around May, in #15.

**Comment by goba** [comment]
Doing this update would be great for many reasons, including the ability to coexist with Upgrade Status in Drupal 8 :) The recent phpunit update in core allows rector to be installable as part of a Drupal project with a Drupal UI, and I am looking at integrating it with upgrade_status at https://www.drupal.org/project/upgrade_status_rector :)
